cwl/facets-suite-workflow.cwl

```diff
@@ -426,6 +426,21 @@ steps:
     out:
       [ output_file, failed_txt, stdout_txt, stderr_txt ]
 
+  merge_maf:
+    run: update_cBioPortal_data.cwl
+    in:
+      pair_id: pair_id
+      subcommand:
+        valueFrom: ${ return "merge_mafs"; }
+      input_file: check_label_maf_normal/output_file
+      output_filename:
+        valueFrom: $(inputs.pair_id)_hisens.ccf.portal.maf
+      facets_txt: check_run_facets/single_facets_txt
+    scatter: [input_file]
+    scatterMethod: dotproduct
```

Comment:
shouldnt need to scatter this, it should just be 1 portal file vs. 1 facets file, I think